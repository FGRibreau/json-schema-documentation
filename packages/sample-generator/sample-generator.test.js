test('should work, yep', () => {
  const sampleGenerator = require('.');
  const schema = {
    title: 'Person',
    type: 'object',
    properties: {
      firstName: {
        type: 'string',
        example: 'dupont',
      },
      lastName: {
        type: 'string',
        example: 'dupont',
      },
      age: {
        description: 'Age in years',
        type: 'integer',
        minimum: 0,
        example: 10,
      },
    },
    required: ['firstName', 'lastName'],
  };

  sampleGenerator(schema, schema).then(
    result => {
      expect(result).toMatchSnapshot();
    },
    err => {
      throw err;
    }
  );
});
