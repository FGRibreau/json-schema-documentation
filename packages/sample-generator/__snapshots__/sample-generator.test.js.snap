// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should work, yep 1`] = `
Object {
  "age": 10,
  "firstName": "dupont",
  "lastName": "dupont",
}
`;
